autometrics/src/lib.rs

```diff
@@ -186,9 +186,21 @@ pub use autometrics_macros::ResultLabels;
 #[cfg(feature = "prometheus-exporter")]
 pub use self::prometheus_exporter::*;
 
+/// Functionality specific to the libraries used to collect metrics
+pub mod integrations {
```

Comment:
Still not sure I like this name, but I can't think of anything better. `metrics_libraries`, `producers`, ...

Reply:
Hmm that would also be an option. The main downside with that is we may want some other submodules exported from the crate root that are different, and they could get a bit lost. For example #90. We'd have 4-5 modules that are all related to the metrics libraries and 1-2 that aren't.

Reply:
seems fine imo